modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/list-type-definitions/request-behalf.json

```
@@ -0,0 +1,6 @@
+{
+	"name": "GroupBehalf",
```

Substitute Group for Request
```suggestion
	"name": "RequestBehalf",
```